pyaerocom/aeroval/_processing_base.py

```diff
@@ -127,7 +104,7 @@ def get_colocator(self, model_name: str = None, obs_name: str = None) -> Colocat
             mod_cfg = self.cfg.get_model_entry(model_name)
             col_cfg["model_cfg"] = mod_cfg
 
-            # LB: Hack and at what lowlevel_helpers's import_from was doing
+            # Hack and at what lowlevel_helpers's import_from was doing
```

Comment:
I don't understand this comment

Reply:
This has been in the code for many months now, just removing my initials. It's saying that the following lines of code were taken from a function which was broken apart.